### ISSUE TITLE
Wrong CMake tests on CudaCommon_SOURCE_DIR

### ISSUE THREAD

**Issue by SimonRit** [RTKConsortium/RTK#655]
RTK code tests CudaCommon_SOURCE_DIR at two lowations since #617 :
https://github.com/RTKConsortium/RTK/blob/01822228a8edf9be587574ef36e27af6fcee537f/wrapping/itkCudaImageRTK.wrap#L3
https://github.com/RTKConsortium/RTK/blob/01822228a8edf9be587574ef36e27af6fcee537f/rtkConfiguration.h.in#L40
That does not seem to have the expected effect as
- I'm not able to compile the Python wrappings with the error
```
CMake Error: File /home/srit/src/itk/itk/Modules/Remote/CudaCommon/wrapping/CudaImage.i.init does not exist.
```
- There are compilation errors on the dashboard, see e.g. [here](https://my.cdash.org/viewBuildError.php?buildid=2756106)

**Comment by SimonRit** [comment]
Do you mind taking a look @LucasGandel?

**Comment by SimonRit** [comment]
Fixed by #659
